.nvmrc

```
@@ -1 +1 @@
-18.13.0
\ No newline at end of file
+20.11.1
```

```suggestion
20
```
Should just work right?

---

Yepp, think it should work.